environment.yml

```diff
@@ -15,4 +15,4 @@ dependencies:
   - progressbar2
   - pytorch
   - pip:
-    - ramp-workflow
+    - ramp-workflow<0.5.0
```

Comment:
please revert this as we will need the latest version on the server to deploy the challenge in the coming days. This ramp-kit must work with the latest ramp-workflow on the pypi